server/odc-service/src/main/java/com/oceanbase/odc/service/sqlcheck/rule/MySQLAffectedRowsExceedLimit.java

```diff
@@ -86,11 +83,10 @@ public long getStatementAffectedRows(Statement statement) {
                                     : getMySqlAffectedRowsByExplain(explainSql, this.jdbcOperations);
                             break;
                         case OB_MYSQL:
-                            affectedRows = getOBMySqlAffectedRows(explainSql, this.jdbcOperations);
+                            affectedRows = getOBAffectedRows(statement.getText(), this.jdbcOperations);
                             break;
                         default:
-                            log.warn("Unsupported dialect type: {}", this.dialectType);
-                            break;
+                            throw new UnsupportedOperationException("Unsupported dialect type: " + this.dialectType);
```

Comment:
1. which caller will catch this exception?

Reply:
For example,  when constructing `MySQLAffectedRowsExceedLimit `object, the member field DialectType assign into `DialectType.ORACLE`